ethash/src/compute.rs

```diff
@@ -85,6 +86,7 @@ pub type H256 = [u8; 32];
 pub struct Light {
 	block_number: u64,
 	cache: Vec<Node>,
+	eth_compute: Mutex<EthCompute>,
```

Comment:
Ah I see. I need the Mutex somewhere (rustc tells me so!), but yes I can move it up the structure a bit.